src/clj_github/repository.clj

```diff
@@ -74,7 +75,7 @@
   Look at https://developer.github.com/v3/repos/branches/#get-a-branch for details about the response format."
   [client org repo branch]
   (fetch-body! client {:method :get
-                       :path (format "/repos/%s/%s/branches/%s" org repo branch)}))
+                       :path (format "/repos/%s/%s/branches/%s" org repo (encode/form-encode branch))}))
```

Comment:
Nice! I didn't know this library. 
When reading the docs available, I saw this function [url-encode](http://ring-clojure.github.io/ring-codec/ring.util.codec.html#var-url-encode) Did you consider using it instead of the `form-encode`? Isn't it fit better in this request?

Reply:
No, didn't know it existed, i will change it